src/screens/SUE/SueReportScreen.tsx

```diff
@@ -315,8 +315,8 @@ export const SueReportScreen = ({
     );
   }
 
-  if (isDone) {
-    return <SueReportDone navigation={navigation} />;
+  if (isDone || isMutationLoading) {
```

Comment:
the state of `isLoading` can change very fast, which results in a bad user experience. we should at least wait 1.5 or 2 seconds showing the sending status before showing the done.

Reply:
Yes, that's a very good idea. For this I think we can update the state a little late with `setTimeout`.

Reply:
but if we are uploading images, we do not want to have the additional timeout added, because the process is already running for a bit.

Reply:
But it's very difficult to control. Because if the image size is 100Kb, the report will still be sent very quickly. Maybe we can add size control 🤷‍♂️

Reply:
I added a 1.5 second timer.

Simulator Screen Recording - iPhone 15 Pro Max - 2024-02-13 at 18.15.33.mp4
https://github.com/smart-village-solutions/smart-village-app-app/pull/1023/commits/f3f860f5592136f19511931388343c6bdd677dd3